src/Microsoft.ML.Data/Properties/AssemblyInfo.cs

```diff
@@ -13,6 +13,7 @@
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Predictor.Tests" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.TimeSeries.Tests" + PublicKey.TestValue)]
 
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Benchmarks" + PublicKey.TestValue)]
```

Comment:
>Benchmarks [](start = 58, length = 10)

Out of curiosity, how are the benchmarks using this? Direct testing of seekable, other...?

Reply:
Yes a benchmark test `CacheDataViewBench`; specifically,
```csharp
            _col = _cacheDataView.Schema.GetColumnOrNull("A").Value;
            _seeker = ((IRowSeekable)_cacheDataView).GetSeeker(colIndex => colIndex == _col.Index);
            _seekerGetter = _seeker.GetGetter<int>(_col.Index);
```

---
In reply to: [249226867](https://github.com/dotnet/machinelearning/pull/2192#discussion_r249226867) [](ancestors = 249226867)

Reply:
Ah. Yes that's fair, we would want to test this directly.

---
In reply to: [249872817](https://github.com/dotnet/machinelearning/pull/2192#discussion_r249872817) [](ancestors = 249872817,249226867)